src/main/java/Duke.java

```diff
@@ -1,10 +1,94 @@
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo =
+                " __        _        \n"
+                        + "|  _ \\ _   | | __ \n"
+                        + "| | | | | | | |/ / _ \\\n"
+                        + "| || | || |   <  __/\n"
+                        + "|_/ \\,||\\\\___|\n";
+        System.out.println("Hi Im Duke, how may I help you?");
+        while(true) {
+            try {
+                String cmd = sc.next();
+                if (cmd.equals("bye")) {
+                    byeUser(logo);
+                    break;
+                } else if (cmd.equals("list")) {
+                    listItems();
+                } else if (cmd.equals("done")) {
+                    int itemNo = sc.nextInt();
+                    markItemAsDone(itemNo);
+                } else if(cmd.equals("delete")) {
+                    int itemNo = sc.nextInt();
+                    deleteItem(itemNo);
+                } else {
+                    String typeOfEvent = cmd;
+                    String eventDescription = sc.nextLine();
+                    addItem(typeOfEvent, eventDescription);
+                }
+            } catch (InvalidCommandException e) {
+                System.out.println(e.getMessage());
+            } catch (EmptyArgumentException e) {
+                System.out.println(e.getMessage());
+            }
+        }
     }
+
+    private static void addItem(String typeOfEvent, String eventDescription) throws EmptyArgumentException, InvalidCommandException {
+        if(typeOfEvent.equals("todo")) {
+            tasks.add(new ToDos(eventDescription));
+        }
+        else if(typeOfEvent.equals("deadline") || typeOfEvent.equals("event")){
+            String[] _eventDescription = eventDescription.split("/");
+            StringBuilder firstPart = new StringBuilder(_eventDescription[0]).append(" ");
+            StringBuilder secondPart = new StringBuilder(_eventDescription[1]);
+            secondPart.insert(0, '(');
+            secondPart.insert(3, ':');
+            secondPart.append(')');
+            firstPart.append(secondPart);
+            if (typeOfEvent.equals("deadline")) tasks.add(new Deadline(eventDescription));
+            else if (typeOfEvent.equals("event")) tasks.add(new Event(firstPart.toString()));
+        }
+        else throw new InvalidCommandException();
+        System.out.print("added: ");
+        System.out.println(tasks.get(tasks.size()-1));
+        System.out.println("Now you have " + tasks.size() + " tasks in the list");
+    }
+
+    private static void markItemAsDone(int itemNo) {
+        tasks.get(itemNo-1).isCompleted = true;
```

Comment:
There should spaces around operators. e.g. `tasks.get(itemNo - 1).isCompleted = true;`